File: app/src/main/java/com/user/exoplayer/player/data/database/AppDatabase.java
package com.user.exoplayer.player.data.database;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

@Database(entities = {VideoUrl.class, Subtitle.class} , version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract UrlDao urlDao();

}

File: app/src/main/java/com/arezoonazer/videoplayer/data/database/AppDatabase.kt
package com.arezoonazer.videoplayer.data.database

import android.arch.persistence.room.Database
import android.arch.persistence.room.Room
import android.arch.persistence.room.RoomDatabase
import android.content.Context

@Database(entities = [Video::class, Subtitle::class], version = 1)
abstract class AppDatabase : RoomDatabase() {
    abstract fun videoDao(): VideoDao

    companion object {
        private var INSTANCE: AppDatabase? = null
        private val sLock = Any()

        fun getDatabase(context: Context): AppDatabase {
            synchronized(sLock) {
                if (INSTANCE == null) {
                    INSTANCE = Room.databaseBuilder(context.applicationContext,
                            AppDatabase::class.java, "PLAYER_DB")
                            .allowMainThreadQueries()
                            .fallbackToDestructiveMigration()
                            .build()
                }
                return INSTANCE!!
            }
        }
    }


}
